awscli/alias.py

```diff
@@ -248,21 +259,25 @@ def _get_alias_args(self):
         except ValueError as e:
             raise InvalidAliasException(
                 'Value of alias "%s" could not be parsed. '
-                'Received error: %s when parsing:\n%s' % (
-                    self._alias_name, e, self._alias_value)
+                'Received error: %s when parsing:\n%s'
+                % (self._alias_name, e, self._alias_value)
```

Comment:
nit; this should be an f-string (or at the least `.format()`). We've generally been moving away from the % syntax.